### ISSUE TITLE
Expand applicability of Step1 to a broader range of identifiers

### ISSUE THREAD

**Issue by greenetarget** [jclachance/BOFdat#36]
I downloaded a xml- model constructed by others, the file is using SEED ID for metabolite and when I ran DNA/RNA/protein_coefficients, it gives:
AttributeError: DictList has no attribute or entry atp_c
AttributeError: 'dict' object has no attribute 'iteritems'

Any suggestion?... Thank you!

**Comment by jclachance** [comment]
Yes, BOFdat is implemented for BiGG identifiers. The SEED identifiers in your model will therefore not be recognized. 

It was previously suggested to add a conversion feature so that other identifiers could be used, which would indeed greatly improve the applicability of the package. I am changing the name of that issue to refer specifically to this issue.

**Comment by hariszaf** [comment]
Hi @jclachance . 
It seems that you had merged @EBosi  PR in the main branch but the `conversion_table` is not there.. 
I thought of letting you know as it 's a bit weird to me. 
As changes came through years, I have a revised version of his script if you d be interested.
